Descriptors/TribalAffiliationDescriptor.xml

```diff
@@ -2808,6 +2790,12 @@
 		<Description>Seminole Tribe of Florida</Description>
 		<Namespace>uri://ed-fi.org/TribalAffiliationDescriptor</Namespace>
 	</TribalAffiliationDescriptor>
+	<TribalAffiliationDescriptor>
+		<CodeValue>Seminole Nation  of Oklahoma</CodeValue>
+		<ShortDescription>Seminole Nation  of Oklahoma</ShortDescription>
```

Comment:
Extra space <CodeValue>Seminole Nation'  'of Oklahoma</CodeValue>
		<ShortDescription>Seminole Nation'  'of Oklahoma

Reply:
![image](https://user-images.githubusercontent.com/83614639/162016291-26699115-b4ca-4639-ba8e-a8f5b0d129b2.png)
I can't find the double space on my end